src/type-layout.md

```diff
@@ -197,6 +197,8 @@ Here is this algorithm described in pseudocode.
 
 <!-- ignore: pseudocode -->
 ```rust,ignore
+// Returns the amount of padding needed after `offset` to ensure that the
+// following address will be aligned to `alignment`.
```

Comment:
Please use `///`

Reply:
I figured it wouldn't matter in pseudocode, but I'll do it.

Reply:
I figured if we are doing this we might as well do it right. ;)

Reply:
Fair enough, haha. At this point I wonder if this can even be called pseudocode except by virtue of technically not being executable (keyword collision, implicit struct/field definitions), but the merits of using the pseudo-Rust syntax are beyond the scope of this PR. :P